tests/S-7_Get_Grades.test.js

```diff
@@ -0,0 +1,137 @@
+// This file contains tests for the following
+// API :
+//      GET     /user/{userName}/class/{className}/grade
+
+const test = require('ava')
+const app = require('../index.js')
+const http = require('http')
+const listen = require('test-listen')
+const got = require('got')
+const onlyFunc = require('../service/DefaultService.js');
+
+// Initialize server
+test.before(async (t) => {
+  console.log('Creating test server')
+  t.context.server = http.createServer(app)
+  t.context.prefixUrl = await listen(t.context.server)
+  t.context.got = got.extend({ prefixUrl: t.context.prefixUrl, responseType: 'json' })
+  console.log('Successfully created test server')
+})
+
+// Shutdown server
+test.after.always(async (t) => {
+  console.log('Testing finished, closing test server')
+  await t.context.server.close()
+  console.log('Successfully closed test server')
+})
+
+// Test the function "getUserGrade" that gets called when the endpoint is used
+test('getUserGrade resolves when called with a userName and className', async (t) => {
+  const userName = 'userName'
+  const className = 'className' 
+  
+  const body = await onlyFunc.getUserGrade( userName , className );
+
+  console.log(body)
+    
+  testValidClass(t, body, className)
+  testValidUser(t,body, userName)
+  testValidGrade(t, body)
+})
+
+// Test for:    GET     /user/{userName}/class/{className}/grade
+test('GET endpoint user/{userName}/class/{className}/grade', async (t) => {
+  const userName = 'userName'
+  const className = 'className' 
+  
+  const { statusCode, body} = await t.context.got.get(`user/${userName}/class/${className}/grade`)
```

Comment:
why you are using template literals instead of string?

**Template literals** are literals delimited with backtick (\`) characters, allowing for [multi-line strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings), [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) with embedded expressions, and special constructs called [tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates).
Example: const userName = \` this is my ${name}\`  where name is replaced by a variable called name.

The **String** object is used to represent and manipulate a sequence of characters. Is used by placing single or double quote between the text you need as a string (`'` or `"`). There are no variables such as template literals.
Example: const userName = 'this is my name' where name has to be replaced by using a javascript function like `.replace()`